meta-oe/recipes-graphics/tigervnc/tigervnc_1.10.1.bb

```diff
@@ -3,7 +3,7 @@ HOMEPAGE = "http://www.tigervnc.com/"
 LICENSE = "GPLv2+"
 SECTION = "x11/utils"
 DEPENDS = "xserver-xorg gnutls jpeg libxtst gettext-native fltk"
-RDEPENDS_${PN} = "coreutils hicolor-icon-theme perl"
+RDEPENDS_${PN} = "coreutils hicolor-icon-theme perl xkbcomp"
```

Comment:
We need similar patch on master and kirkstone too. I have cherry-picked it form master-next and if all is well it will land in master soon FYI @akuster

Reply:
applied to master with 447de4d47ba2deba1af80201b91bb312f184fe0e